app/boards/arm/bluemicro840/bluemicro840_v1_defconfig

```diff
@@ -10,6 +10,8 @@ CONFIG_ARM_MPU=y
 # enable GPIO
 CONFIG_GPIO=y
 
+CONFIG_ADC=y
```

Comment:
Instead of just forcing this this way, you should have a new Kconfig setting for this new driver, and that that driver `depends on ADC`